scheds/rust/scx_p2dq/src/bpf/main.bpf.c

```diff
@@ -761,6 +793,91 @@ void BPF_STRUCT_OPS(p2dq_stopping, struct task_struct *p, bool runnable)
 	}
 }
 
+static bool dispatch_cpu(u64 dsq_id, s32 cpu, struct llc_ctx *llcx, int dsq_index)
```

Comment:
nit: do you want either `__always_inline` or `__weak` here to prevent the compiler being clever?